### PR TITLE
Docs releasing

### PR DESCRIPTION
**Pull request by ncphillips**
Just updating the docs to better reflect my actual process for releasing.

### CODE REVIEW

**Comment by ncphillips** [comment]
Unrelated but

@all-contributors please add @weibenfalk for video

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,10 +68,14 @@ Linking prevents running `npm install` from directly inside a package from worki
...
    This sucks. Try running `lerna clean` and then running `npm run bs` again.
 
-## Releasing
+## General Release Process
 
 The general release process looks like this:
 
+1. **Merge Changes**
+
+   Merge the changes to be published into the appropriate branche.
```
Comment (dwalkr) [inline CONTRIBUTING.md]:
```suggestion
   Merge the changes to be published into the appropriate branch.
```

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+   ```
+   git push && git push --tags
+   ```
+
+1. **Backmerge to `master`**
+
+   ```
+   git checkout master
+   git merge next
```
Comment (dwalkr) [inline RELEASE.md]:
```suggestion
   git merge next
   git push
```

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+1) **Push CHANGELOGs and git tags to Github:**
+   ```
+   git push && git push --tags
+   ```
+
+1. **Backmerge to `next`**
+
+   ```
+   git checkout next
+   git merge latest
```
Comment (dwalkr) [inline RELEASE.md]:
```suggestion
   git merge latest
   git push
```

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+   ```
+   lerna publish from-package
+   ```
+
+1) **Push CHANGELOGs and git tags to Github:**
+   ```
+   git push && git push --tags
+   ```
+
+1. **Backmerge to `next`**
```
Comment (DirtyF) [inline RELEASE.md]:
```suggestion
1. **Backport to `next`**
```

Comment (DirtyF) [reply RELEASE.md]: Do you mean backport changes here?

Comment (ncphillips) [reply RELEASE.md]: Essentially, but backmerge is the git specific term. We merge the changes from `latest` back onto `next`

**Comment by ncphillips** [comment]
I'm going to update this again

**Pull request merged**